app/src/main/res/values/strings.xml

```diff
@@ -1654,10 +1655,32 @@
     <string name="patroller_diff_feedback_dialog_option_neutral">Neutral</string>
     <string name="patroller_diff_feedback_dialog_option_unsatisfied">Unsatisfied</string>
     <string name="patroller_diff_feedback_dialog_feedback_title">How can we improve the feature?</string>
+    <string name="patroller_diff_menu_edit_history_title">Article edit history</string>
     <string name="patroller_diff_feedback_dialog_input_hint">Feedback</string>
     <string name="patroller_diff_feedback_dialog_submit">Submit</string>
     <string name="patroller_diff_feedback_submitted_snackbar">Feedback submitted.</string>
     <string name="patroller_diff_feedback_tooltip">Tap on overflow menu to submit feedback anytime under \'Problem with feature\'.</string>
+    <string name="patroller_saved_message_title_1">Vandalism warning</string>
+    <string name="patroller_saved_message_title_2">Editing tests</string>
+    <string name="patroller_saved_message_title_3">Neutral point of view</string>
+    <string name="patroller_saved_message_title_4">Automatic translation</string>
+    <string name="patroller_saved_message_title_5">Conflict of interest reminder</string>
+    <string name="patroller_saved_message_title_6">Final warning</string>
+    <string name="patroller_saved_message_title_7">Copyright violation alert</string>
+    <string name="patroller_saved_message_title_8">Leave an edit summary</string>
+    <string name="patroller_saved_message_title_9">Do not censor</string>
+    <string name="patroller_saved_message_title_10">Article improvement suggestion</string>
+    <string name="patroller_saved_message_1">Hi {{{username}}}, I want to let you know I\'ve reverted one or more of your [{{{diffLink}}}|recent contributions] because they do not seem constructive. If you think I made a mistake or have questions, you can leave a message on [[User talk:{{{senderUserName}}}|my talk page]]. Please note that continued vandalism may lead to restrictions.</string>
+    <string name="patroller_saved_message_2">Hi {{{username}}}, I am delighted that Wikipedia has caught your attention and that you have been interested in learning how to edit on it. The editing test you did worked; However, you should keep in mind that the goal of this project is to develop a reliable and complete encyclopedia that everyone can consult. Your edit affected the content, so it has been removed.\nIf you want to learn how to edit Wikipedia, the [[Wikipedia:Sandbox|sandbox]]  is at your disposal. If your comments were specific to an article, the correct place is that article\'s talk page.</string>
```

Comment:
If a messages requires a paragraph break, it needs to be two newline characters (`\n\n`) instead of one.